### DISCUSSION TITLE
how to set background-color dynamically

### DISCUSSION THREAD

**Discussion by surushTodzhibekov** [quasarframework/quasar#11168] [General - Components / Directives / etc] [answered]
I want to set my background various color from database but it does not work 
`<q-page class="q-pa-sm">
    <q-card
      v-for="book in books"
      :key="book.id"
      clickable
      @click="onClick(book.id)"
      class="my-card q-mb-sm"
      flat
      bordered
    >
      <q-card-section horizontal>
        <q-card-section class="page-card" style="background-color: book.bg">
          <div class="text-category text-white">{{ book.name }}</div>
          <q-btn to="/list" class="text-secondary" flat round>
            
            <img v-bind:src="book.image_path" class="img-category" />
          </q-btn>
        </q-card-section>
      </q-card-section>
    </q-card>
  </q-page>`
this is my code 
`"data": [
      {
        "tableName": "books",
        "inbound": true,
        "rows": [
          {
            "name": "Кодекси андоз",
            "image_path": "Kodeksi-gumruk.png",
            "language": "tj",
            "bg": "#3075FF",
            "id": 1
          },
          {
            "name": "Кодекси гумрук",
            "image_path": "nalogovoy-kodeks1.png",
            "language": "tj",
            "bg": "#1C3AA9",
            "id": 2
          }, ]]`
the above code from database
![bg](https://user-images.githubusercontent.com/38825951/139571978-12ed7adb-9e5c-4c82-8bba-692356056d5f.png)
it should look like this 
![bg1](https://user-images.githubusercontent.com/38825951/139572029-e12cf475-e375-4f2e-9e92-2dfd6883943b.png)
I appreciate any helps

**Comment by hawkeye64** [comment, marked as answer]
It's this line, because the browser is treating as text, but you need it translated from JavaScript. So, you need to let Vue know it needs JavaScript treatment.

Current line:
```js
style="background-color: book.bg"
```

Change to:
```js
:style="{ backgroundColor: book.bg }"
```

**Reply by surushTodzhibekov** [reply]
Tnak you so much. from morning until now I can't fixed it. By your help I fixed my issue.
